infra/scripts/local-ci.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/bash
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+##### defaults
+secretsfile=""
+docker_test="Group1-Docker-Commands"
+target_vch=""
+odir="ci-results"
+ci_container="gcr.io/eminent-nation-87317/vic-integration-test:1.39"
+github_api_key=""
+test_url=""
+test_datastore=""
+test_username=""
+test_password=""
+vic_dir=$(pwd)
```

Comment:
This is used to locate test sources so be explicit by basing of path of the script in the repo:
```
BASE_DIR=$(dirname $(readlink -f "$BASH_SOURCE"))
vic_dir=${BASE_DIR}/../../
```